packages/app/src/search/dto/search-result.object.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { DataIndex } from "App/index-storage/constants"
+
+type ScopeKeys =
+  | DataIndex.Contact
+  | DataIndex.Message
+  | DataIndex.Template
+  | DataIndex.Thread
+
+export type SearchResult = {
+  // AUTO DISABLED - fix me if you like :)
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key in ScopeKeys]?: any[] | undefined
```

Comment:
```suggestion
  [key in ScopeKeys]?: unknown[] | undefined
```

it's possible to add an easily `uknown` or generic type for this case? 🤔